### ISSUE TITLE
Error when using coordinate arguments in analyze.py: "ValueError: vector too long" 

### ISSUE THREAD

**Issue by Cdevenish** [kahst/BirdNET-Analyzer#248]
I've set up BirdNET in a conda env. When I run the example given on github without coordinate arguments, it works fine. 

`python analyze.py --i example/ --o example/`

But when I add in coordinates:
`python analyze.py --i example/ --o example/ --lat 42.5 --lon -76.45 --week 4 --sensitivity 1.0`

I get "ValueError: vector too long" when it tries to make the species list:

 ```
cfg.SPECIES_LIST = species.getSpeciesList(cfg.LATITUDE, cfg.LONGITUDE, cfg.WEEK, cfg.LOCATION_FILTER_THRESHOLD)
BirdNET-Analyzer\species.py", line 30, in getSpeciesList
  pred = model.explore(lat, lon, week)
BirdNET-Analyzer\model.py", line 481, in explore
  l_filter = predictFilter(lat, lon, week)
BirdNET-Analyzer\model.py", line 455, in predictFilter
  loadMetaModel()
BirdNET-Analyzer\model.py", line 114, in loadMetaModel
  M_INTERPRETER.allocate_tensors()
\envs\birdNET\lib\site-packages\tensorflow\lite\python\interpreter.py", line 513, in allocate_tensors
   return self._interpreter.AllocateTensors()
```

I'm using BirdNET in a conda environment on Windows 10.  python 3.10.13 tensorflow 2.10.0 librosa 0.10.1 resampy 0.4.2

If I use the same code with the packaged exec for windows with coordinates, it also works fine.  e.g.
`BirdNET-Analyzer.exe --i example --o example --lat 53.00153 --lon -2.27406 --week 38 --sf_thresh 0.03`

Thanks for all the work on BirdNET!  :)

**Comment by Cdevenish** [comment]
Thanks. That does fix the above problem, using tensorflow 2.15. 

I do now get a warning:
```
WARNING:tensorflow:
.conda\envs\birdNET\Lib\site-packages\keras\src\losses.py:2976: The name tf.losses.sparse_softmax_cross_entropy is deprecated. Please use tf.compat.v1.losses.sparse_softmax_cross_entropy instead.

Attempting to use a delegate that only supports static-sized tensors with a graph that has dynamic-sized tensors (tensor#187 is a dynamic-sized tensor).
```
But the example does run, with coordinates.

I'm now running in a conda environment on Windows 10. python 3.11.0  tensorflow 2.15.0 librosa 0.10.1 resampy 0.4.3

**Comment by Josef-Haupt** [comment]
If it's working properly, you can just ignore the warnings ;)
